neo/public/javascripts/cache/services.js

```diff
@@ -0,0 +1,356 @@
+ angular.module('cloudberry.cache', ['leaflet-directive', 'cloudberry.common' ])
```

Comment:
Nope this is the naming convention followed by earlier developers and also in Angular JS developers.In the neo/public/javascripts/common they have named it as services.js